WooCommerce/Classes/ViewRelated/Products/Edit Product/EditCategories/ProductCategoryListViewModel.swift

```diff
@@ -34,17 +59,28 @@ final class ProductCategoryListViewModel {
         return resultController.object(at: indexPath)
     }
 
-    /// Load existing categories from storage and fire the synchronize product categories action
+    /// Load existing categories from storage and fire the synchronize all categories action.
     ///
     func performInitialFetch() {
```

Comment:
Should we rename `performInitialFetch` now? It's not the "initial" anymore, it's the "only" fetch I think. 🙂

Reply:
https://github.com/woocommerce/woocommerce-ios/pull/2130/commits/84d5a1b0e4ce79c29baf9ecbd6fb004b8c91c277